From b63dbb5f6c06bebfb03e351262cc96f478ebc967 Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Sat, 2 Nov 2024 11:30:54 +0530
Subject: [PATCH 081/N] Fix include naming for cross build

The casing and pathseps have to match, so a cross compiler
on Linux find the files.
---
 Modules/posixmodule.c  | 2 +-
 Modules/socketmodule.c | 2 +-
 PC/_testconsole.c      | 2 +-
 PC/_wmimodule.cpp      | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/Modules/posixmodule.c b/Modules/posixmodule.c
index b1e23c2..dcc1f4c 100644
--- a/Modules/posixmodule.c
+++ b/Modules/posixmodule.c
@@ -9345,7 +9345,7 @@ os_setpgrp_impl(PyObject *module)
 
 #ifdef MS_WINDOWS
 #include <winternl.h>
-#include <ProcessSnapshot.h>
+#include <processsnapshot.h>
 
 // The structure definition in winternl.h may be incomplete.
 // This structure is the full version from the MSDN documentation.
diff --git a/Modules/socketmodule.c b/Modules/socketmodule.c
index 687594a..bf43175 100644
--- a/Modules/socketmodule.c
+++ b/Modules/socketmodule.c
@@ -279,7 +279,7 @@ shutdown(how) -- shut down traffic in one or both directions\n\
 # endif
 
 /* Helpers needed for AF_HYPERV */
-# include <Rpc.h>
+# include <rpc.h>
 
 /* Macros based on the IPPROTO enum, see: https://bugs.python.org/issue29515 */
 #ifdef _MSC_VER
diff --git a/PC/_testconsole.c b/PC/_testconsole.c
index 0dcea86..c38378a 100644
--- a/PC/_testconsole.c
+++ b/PC/_testconsole.c
@@ -140,7 +140,7 @@ _testconsole_read_output_impl(PyObject *module, PyObject *file)
 }
 
 
-#include "clinic\_testconsole.c.h"
+#include "clinic/_testconsole.c.h"
 
 PyMethodDef testconsole_methods[] = {
     _TESTCONSOLE_WRITE_INPUT_METHODDEF
diff --git a/PC/_wmimodule.cpp b/PC/_wmimodule.cpp
index 58cb9a8..9296d33 100644
--- a/PC/_wmimodule.cpp
+++ b/PC/_wmimodule.cpp
@@ -14,9 +14,9 @@
 #endif
 
 #define _WIN32_DCOM
-#include <Windows.h>
+#include <windows.h>
 #include <comdef.h>
-#include <Wbemidl.h>
+#include <wbemidl.h>
 #include <propvarutil.h>
 
 #include <Python.h>
